test/runtests.jl

```diff
@@ -13,15 +13,29 @@ else
     typemax(Csize_t)
 end
 
+
+test_names = copy(tests)
```

Comment:
It seems it would be cleaner to maintain only one array `tests` instead of two in parallel, and when the name is needed, call a function `test_name` on it which implements the transormation below. Or would it make things more complicated somewhere? (or the other way around, maintaining only `test_names` and have a function `test_path` or something, cf. my next comment below).

Reply:
Or maybe an array of pairs `name => path` or a `Dict` of that.